stanford_profile_helper.module

```diff
@@ -392,3 +392,23 @@ function _stanford_profile_helper_get_lockup_states(array $opts, $input) {
 function stanford_profile_helper_form_config_pages_lockup_settings_form_alter_submit(array &$form, FormStateInterface $form_state) {
   Cache::invalidateTags(['config:system.site']);
 }
+
+/**
+ * Implements hook_preprocess_pattern_NAME().
+ */
+function stanford_profile_helper_preprocess_pattern_alert(&$variables) {
+  $entity_type = $variables['context']->getProperty('entity_type');
+  $bundle = $variables['context']->getProperty('bundle');
+  $entity = $variables['context']->getProperty('entity');
+
+  // Global Messages!
+  if ($entity_type == "config_pages" && $bundle == "global_message") {
+    $color = $entity->get('su_global_msg_type')->getString();
```

Comment:
I prefer to always check if the entity has the field first. `$entity->hasField('su_global_msg_type')`

Reply:
Yeah, I suppose this would be a bad place to get a 500 error.
I'll add the check and log a warning if it is missing.